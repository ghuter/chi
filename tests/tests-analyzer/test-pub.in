pub f :: fun () i32 {
	return 0;
}

pub Point :: struct {
	x: i32;
	y: i32;
}
