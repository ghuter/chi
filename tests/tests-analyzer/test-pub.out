SSTRUCT(Point, x : i32, y : i32)
SFUN(f, {SRETURN(ECSTI(0))}) : i32
